lib/change_stream.js

```diff
@@ -267,8 +267,8 @@ var createChangeStreamCursor = function(self) {
    * @event ChangeStream#error
    * @type {Error}
    */
-  changeStreamCursor.on('error', function(error) {
-    self.emit('error', error);
+  changeStreamCursor.once('error', function(error) {
+    processNewChange(self, error, null);
```

Comment:
This allows the stream to resume, but no `'data'` events are received after the resume ([L239](https://github.com/mongodb/node-mongodb-native/blob/master/lib/change_stream.js#L239) is not called). I'm not sure why this happens, but I'll continue to investigate tomorrow.